pkg/scheduler/context.go

```diff
@@ -631,6 +635,23 @@ func (cc *ClusterContext) updateNode(nodeInfo *si.NodeInfo) {
 
 	switch nodeInfo.Action {
 	case si.NodeInfo_UPDATE:
+		var newReadyStatus bool
+		var err error
+		if newReadyStatus, err = strconv.ParseBool(nodeInfo.Attributes["ready"]); err != nil {
+			log.Logger().Error("Could not parse ready attribute, assuming true", zap.Any("attributes", nodeInfo.Attributes))
+			newReadyStatus = true
```

Comment:
Are we sure that we get this information from the k8shim?
Looking at what is send by the shim we seem to only get the resources and some fixed rack related info

Reply:
Please check https://github.com/apache/incubator-yunikorn-k8shim/pull/362. This is the shim side change.